tests/ssl-opt.sh

```diff
@@ -1120,6 +1125,11 @@ run_test() {
         return
     fi
 
+    if is_excluded_test_suite "${TEST_SUITE_NAME:-ssl-opt}"; then
```

Comment:
It's not very clear why any test that doesn't have a name is substituted with "ssl-opt" (Related to my other comments, it's not even that clear that this is the operation that's happening), so a comment would be useful.